src/scales/categoryScale.ts

```diff
@@ -3,7 +3,6 @@
 module Plottable {
 export module Scales {
   export class Category extends Scale<string, number> {
-    protected _d3Scale: D3.Scale.OrdinalScale;
```

Comment:
Is there a good reason why this was deleted? I see two reasons to keep it:
- When reading code, you don't have to navigate to a different file to find the variable
- You don't need those ugly casts (which get uglier in the next files) every time you want to use the scale.

If we decide to keep it, this comment applies to all future files (including those where the `protected _d3Scale` was not present in the first place (for consistency)